From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Fri, 22 Apr 2016 00:42:21 -0400
Subject: add syscfg alias for powerpc-unknown-linux-gnuspe

See
https://buildd.debian.org/status/fetch.php?pkg=libgpg-error&arch=powerpcspe&ver=1.21-2&stamp=1458593907,
which produces the same header as powerpc-unknown-linux-gnu
---
 src/mkheader.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/mkheader.c b/src/mkheader.c
index 0d3d6ba..64d0904 100644
--- a/src/mkheader.c
+++ b/src/mkheader.c
@@ -82,6 +82,7 @@ canon_host_triplet (const char *triplet)
     {"i486-pc-kfreebsd-gnu", "i686-pc-kfreebsd-gnu"},
     {"i586-pc-kfreebsd-gnu"},
     {"x86_64-pc-linux-gnuhardened1", "x86_64-pc-linux-gnu" },
+    {"powerpc-unknown-linux-gnuspe", "powerpc-unknown-linux-gnu" },
 
     { NULL }
   };
